From 6e2aadfe7cd5b3e69f5d21c95bf022eed07ec165 Mon Sep 17 00:00:00 2001
From: Robby Cai <R63905@freescale.com>
Date: Fri, 3 Dec 2010 10:20:07 +0800
Subject: [PATCH] ENGR00134277 MX50: turn off the ZQ calibration config

The dynamic ZQ calibration procedure need some revision and further debug.
Currently turn off this option for safety.

Signed-off-by: Robby Cai <R63905@freescale.com>
---
 arch/arm/configs/imx5_defconfig |   14 ++++++++++++--
 1 files changed, 12 insertions(+), 2 deletions(-)

diff --git a/arch/arm/configs/imx5_defconfig b/arch/arm/configs/imx5_defconfig
index e19d9db..63abbc5 100644
--- a/arch/arm/configs/imx5_defconfig
+++ b/arch/arm/configs/imx5_defconfig
@@ -1,6 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux kernel version: 2.6.35.3
+# Fri Dec  3 09:49:37 2010
 #
 CONFIG_ARM=y
 CONFIG_HAVE_PWM=y
@@ -90,6 +91,7 @@ CONFIG_SIGNALFD=y
 CONFIG_TIMERFD=y
 CONFIG_EVENTFD=y
 CONFIG_SHMEM=y
+# CONFIG_ASHMEM is not set
 CONFIG_AIO=y
 CONFIG_HAVE_PERF_EVENTS=y
 CONFIG_PERF_USE_VMALLOC=y
@@ -272,7 +274,7 @@ CONFIG_MXC_PWM=y
 CONFIG_ARCH_HAS_RNGC=y
 CONFIG_ARCH_MXC_IOMUX_V3=y
 CONFIG_MXC_DVFS_PER=y
-CONFIG_MXC_ZQ_CALIBRATION=y
+# CONFIG_MXC_ZQ_CALIBRATION is not set
 
 #
 # Processor Type
@@ -411,10 +413,14 @@ CONFIG_HAVE_AOUT=y
 #
 CONFIG_PM=y
 CONFIG_PM_DEBUG=y
-CONFIG_SUSPEND_DEVICE_TIME_DEBUG=y
+# CONFIG_PM_ADVANCED_DEBUG is not set
+# CONFIG_PM_VERBOSE is not set
+CONFIG_CAN_PM_TRACE=y
 CONFIG_PM_SLEEP=y
 CONFIG_SUSPEND_NVS=y
 CONFIG_SUSPEND=y
+# CONFIG_PM_TEST_SUSPEND is not set
+CONFIG_SUSPEND_DEVICE_TIME_DEBUG=y
 CONFIG_SUSPEND_FREEZER=y
 CONFIG_APM_EMULATION=y
 CONFIG_PM_RUNTIME=y
@@ -694,6 +700,8 @@ CONFIG_MISC_DEVICES=y
 # CONFIG_SENSORS_TSL2550 is not set
 # CONFIG_DS1682 is not set
 # CONFIG_TI_DAC7512 is not set
+# CONFIG_ANDROID_PMEM is not set
+# CONFIG_UID_STAT is not set
 # CONFIG_C2PORT is not set
 
 #
@@ -834,6 +842,7 @@ CONFIG_SMSC911X=y
 # CONFIG_KS8851 is not set
 # CONFIG_KS8851_MLL is not set
 CONFIG_FEC=y
+# CONFIG_FEC_1588 is not set
 # CONFIG_FEC2 is not set
 # CONFIG_NETDEV_1000 is not set
 # CONFIG_NETDEV_10000 is not set
@@ -1641,6 +1650,7 @@ CONFIG_SND_SOC_IMX_3STACK_SGTL5000=y
 # CONFIG_SND_SOC_IMX_3STACK_AK5702 is not set
 # CONFIG_SND_SOC_IMX_3STACK_BLUETOOTH is not set
 CONFIG_SND_SOC_IMX_3STACK_CS42888=y
+# CONFIG_MXC_SSI_DUAL_FIFO is not set
 CONFIG_SND_SOC_I2C_AND_SPI=y
 # CONFIG_SND_SOC_ALL_CODECS is not set
 CONFIG_SND_SOC_CS42888=y
-- 
1.5.4.4

